realtime/users.go

```diff
@@ -60,11 +61,23 @@ func (c *Client) Login(credentials *models.UserCredentials) (*models.User, error
 	return getUserFromData(rawResponse.(map[string]interface{})), err
 }
 
+// Auth a user. Using an authentication token to login.
+//
+// https://rocket.chat/docs/developer-guides/realtime-api/method-calls/login/#using-an-authentication-token
+func (c *Client) Auth(token string) (*models.User, error) {
```

Comment:
How about `TokenLogin`? Or merge token access to Login like this logic
```
// Login a user.
// token shouldn't be nil, otherwise the password and the email are not allowed to be nil.
//
// https://rocket.chat/docs/developer-guides/realtime-api/method-calls/login/
func (c *Client) Login(credentials *models.UserCredentials) (*models.User, error) {
	if credentials.Token != "" {

	} else {

	}
}
```

Any other suggestions?

Reply:
That would work even better.  Bit more familiar in terms of go libraries and their behaviors :smile: